compliance_checker/cf/appendix_f.py

```diff
@@ -235,3 +235,33 @@
         )
     ]
 }
+
+grid_mapping_dict17 = grid_mapping_dict16.copy() # need shallow copy; update() returns None
+grid_mapping_dict17.update({
+    'geos': [
```

Comment:
Shouldn't this be `geostationary` per http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html#_geostationary_projection?

Reply:
Ah... yeah. Looks like I typed too quickly and overlooked that